README.md

```diff
@@ -23,6 +23,8 @@ This repository is structured as follows:
 
 ## Installation
 
+> **WARNING**: Even if you install MALA via PyPI, please consult the full installation instructions afterwards. External modules (like the QuantumESPRESSO bindings) are not distributed via PyPI!
```

Comment:
Is this `>` before `**WARNING**` of any value, or just a typo?

Reply:
It's a blockquote, per this suggestion: https://stackoverflow.com/a/55850542

Reply:
You're right.